packages/at_notify_flutter/pubspec.yaml

```diff
@@ -15,14 +15,14 @@ dependencies:
   flutter:
     sdk: flutter
 
-  at_client_mobile: ^3.2.9
-  at_client: ^3.0.59
-  at_commons: ^3.0.46
+  at_client_mobile: ^3.2.10
+  at_client: ^3.0.61
+  at_commons: ^3.0.48
   at_common_flutter: ^2.0.11
-  flutter_local_notifications: ^9.4.0
+  flutter_local_notifications: ^14.1.1
 
 dependency_overrides:
-  package_info_plus: ^2.0.0
+  package_info_plus: ^3.0.0
```

Comment:
Do we still need this override after using `at_client_mobile : 3.2.10` ?

Reply:
No need for that. Thank you!